src/override/rust.cc

```diff
@@ -1,5 +1,6 @@
-#include "../mem/slowalloc.h"
+#define SNMALLOC_NAME_MANGLE(a) snmalloc_##a
 #include "../snmalloc.h"
```

Comment:
With `malloc.cc` included then this header will be included already.

Reply:
I went ahead and removed ../snmalloc.h too then since malloc.cc has it.